doc/src/install.rst

```diff
@@ -8,26 +8,115 @@ Installation
 Psycopg is a PostgreSQL_ adapter for the Python_ programming language. It is a
 wrapper for the libpq_, the official PostgreSQL client library.
 
-The `psycopg2` package is the current mature implementation of the adapter: it
-is a C extension and as such it is only compatible with CPython_. If you want
-to use Psycopg on a different Python implementation (PyPy, Jython, IronPython)
-there is a couple of alternative:
+.. _PostgreSQL: https://www.postgresql.org/
+.. _Python: https://www.python.org/
 
-- a `Ctypes port`__, but it is not as mature as the C implementation yet
-  and it is not as feature-complete;
 
-- a `CFFI port`__ which is currently more used and reported more efficient on
-  PyPy, but please be careful of its version numbers because they are not
-  aligned to the official psycopg2 ones and some features may differ.
+.. index::
+    single: Install; from PyPI
+    single: Install; wheel
+    single: Wheel
 
-.. _PostgreSQL: https://www.postgresql.org/
-.. _Python: https://www.python.org/
-.. _libpq: https://www.postgresql.org/docs/current/static/libpq.html
-.. _CPython: https://en.wikipedia.org/wiki/CPython
-.. _Ctypes: https://docs.python.org/library/ctypes.html
-.. __: https://github.com/mvantellingen/psycopg2-ctypes
-.. __: https://github.com/chtd/psycopg2cffi
+.. _binary-packages:
+
+Quick Install
+-------------
+
+For most operating systems, the quickest way to install psycopg is using the wheel_ package available on PyPI_:
+
+.. code-block:: console
+
+    $ pip install psycopg2-binary
+
+This will install a pre-compiled binary version of the module which does not require the build or runtime prerequisites described below. Make sure to sue an up-date-date version of :program:`pip` (you can upgrade it
+using something like ``pip install -U pip``).
+
+You may then import the ``psycopg`` package, as usual:
+
+.. code-block:: python
+
+    import psycopg
+
+    # Connect to your postgres DB
+    conn = psycopg.connect("dbname=test user=postgres")
+
+    # Open a cursor to perform database operations
+    cur = conn.cursor()
+
+    # Execute a query
+    cur.execute("SELECT * FROM my_data");
+
+    # Retrieve query results
+    records = cur.fetchall()
+
+.. __: PyPI-binary_
+.. _PyPI: https://pypi.org/project/psycopg2-binary/
+.. _wheel: https://pythonwheels.com/
+
+psycopg vs psycopg-binary
```

Comment:
We had a bunch of _Notes_ sections stacked on top of each other. Thought it deserved its own section.

This was also the biggest confusion to me coming in -- why would I want one package over the other?